modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -178,7 +178,7 @@ export async function getAllFolders() {
 
 export async function getDefinitionsByFolderERC(folderErc: string) {
 	const folderDefinitionsResponse = await fetch(
-		`/o/object-admin/v1.0/object-folders/by-external-reference-code/${folderErc}/object-definitions`,
+		`/o/object-admin/v1.0/object-folders/by-external-reference-code/${folderErc}/object-definitions?page=-1`,
```

Comment:
Hey @danvb112, I noticed that you are using the `-1` parameter to get nonpaged values ​​from the API. I believe that for security reasons, in order to avoid `DDOS attacks`. We have to take other approaches to obtain these items.